Source/ProceduralFairings/FairingDecoupler.cs

```diff
@@ -112,8 +112,8 @@ public void FixedUpdate ()
 
                     if (tr)
                     {
-                        part.Rigidbody.AddForce (tr.TransformDirection (forceVector) * Mathf.Lerp (ejectionLowDv, ejectionDv, ejectionPower), ForceMode.VelocityChange);
-                        part.Rigidbody.AddTorque (tr.TransformDirection (torqueVector) * Mathf.Lerp (ejectionLowTorque, ejectionTorque, torqueAmount), ForceMode.VelocityChange);
```

Comment:
Here we have to do it all ourselves.
L=Iω,
τ=L′=Iω′=I∆ω/∆t = ∆L/∆t.
For reference, I am using the following lines of Principia code as a guide to interpret `inertiaTensorRotation`:
https://github.com/mockingbirdnest/Principia/blob/66661d78f531791d2d1e7e3a135ecd13fe30a4f6/ksp_plugin_adapter/ksp_plugin_adapter.cs#L1100-L1108
https://github.com/mockingbirdnest/Principia/blob/48f2466671bb206fc0999da38f36c1eb3b6a6f80/ksp_plugin/interface.cpp#L861-L864

```C#
Vector3 ΔωWorld = tr.TransformDirection (torqueVector) * Mathf.Lerp (ejectionLowTorque, ejectionTorque, torqueAmount);
Vector3 principalMoments = part.RigidBody.inertiaTensorRotation;
Quaternion partPrincipalAxesToWorld = part.RigidBody.inertiaTensorRotation * part.RigidBody.rotation;
Quaternion worldToPartPrincipalAxes = Quaternion.Inverse(partPrincipalAxesToWorld);
Vector3 ΔωPartPrincipalAxes = worldToPartPrincipalAxes * ΔωWorld;
Vector3 ΔLPartPrincipalAxes = Vector3.Scale(principalMoments, ΔωPartPrincipalAxes);
Vector3 ΔLWorld = partPrincipalAxesToWorld * ΔLPartPrincipalAxes;
part.AddTorque(ΔLWorld / TimeWarp.fixedDeltaTime);
```
@pleroy can you double-check that I got the reference frames right here?